### DISCUSSION TITLE
How to get autocompletion for vue components inside markdown files when using vuepress

### DISCUSSION THREAD

**Discussion by Nethrenial** [vuejs/language-tools#978] [Q&A]
# How to get autocompletion for vue components inside markdown files when using vuepress?

I don't know if it's even possible, basically, Can we get IntelliSense(for prop types) and syntax highlighting when using vue components inside Markdown files. For an example in a VuePress project?

**Comment by yaegassy** [comment]
The file types that the "volar" extension works with are `vue` files, and when using "Take Over Mode", additionally `typescript`, `javascript`, `typescriptreact` and `javascriptreact`.

`markdown` files are not supported.

**Comment by johnsoncodehk** [comment]
Please track #820.
